### ISSUE TITLE
Issue with gaussian surface on google chrome 

### ISSUE THREAD

**Issue by giagitom** [molstar/molstar#1187]
Here is what happens on Chrome if I try to rotate the view while changing trajectory frames. My current version is Version 126.0.6478.126 (Official Build) (64-bit)

On Firefox it works fine

https://github.com/molstar/molstar/assets/20627977/d3e7139b-c11a-4cd9-96f9-de592996c0ee

**Comment by arose** [comment]
I looks fine for me playing and rotating/spinning 1NMR. I can't see the video, it seems corrupted.

**Comment by giagitom** [comment]
Here are a few screenshots from the video. I'll check again at the next chrome update. I'm on ubuntu with nvidia card

![image](https://github.com/molstar/molstar/assets/20627977/e3285556-7994-4fe0-9b3f-d6f9cd0109c1)
![image](https://github.com/molstar/molstar/assets/20627977/40b11e34-9720-4786-9fb7-3dc842bfd2a1)
![image](https://github.com/molstar/molstar/assets/20627977/8ae5825f-0ecf-409e-aba2-c156ed526656)

**Comment by arose** [comment]
Could very well be a GPU driver issue. I'd make sure to be on the latest driver.

**Comment by giagitom** [comment]
I just updated to latest drivers but the problem remains for now

**Comment by ValeZAA** [comment]
> I looks fine for me playing and rotating/spinning 1NMR. I can't see the video, it seems corrupted.

The video is not corrupted. You need chrome to play 4:4:4 AVC though.

**Comment by arose** [comment]
related #1208

**Comment by arose** [comment]
@giagitom is this still an issue?

**Comment by giagitom** [comment]
Yes, I'm still seeing this is happening. Something changed though. It is less perceivable during animation but I just noticed that if I just manually change model index (**fast enough**) this strange overlaps, holes, and spikes happens frequently. 

I'm using this to rapidly change model index in the right panel
![image](https://github.com/user-attachments/assets/d38ea643-8bda-47e9-a02e-ef56a043333d)

It seems like canvas is not cleared properly from previous frame, or maybe the computing wasn't finished yet and it outputted an incomplete surface. I can reproduce on Chrome and also on Firefox on both windows and linux machines. It seems something general. 

![image](https://github.com/user-attachments/assets/f4870a7d-d27f-450a-a1bc-c99cbddb54d9)

![image](https://github.com/user-attachments/assets/93cd14eb-2762-4169-a818-eea33650ebe5)

**Comment by giagitom** [comment]
@arose I have just found out that the transparency issue can be observed also on other representations like spheres for example.  (Tested with pdb id 1G03)

![image](https://github.com/user-attachments/assets/f75dfab8-b2dd-47d4-b56a-0dfdc47bcb13)

**Comment by giagitom** [comment]
I am no longer observing the original issue. Still observing sometimes overlaps with previous frame but much less than before and only with surfaces.  I'll reopen if something changes
